### DISCUSSION TITLE
Задание 6

### DISCUSSION THREAD

**Discussion by eprilutskaya** [anastasiaprakapchuk/AmastyLearningNewTechnology#21] [General]
https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/2df301cf33da20d9e475a94159fb4aa6509502d6/SecondAnastasiaModule/Observer/CheckAddProductObserver.php#L45
Чтобы разбить строку по символу и убрать лишние пробелы стоит использовать такую форму
`array_map('trim', explode(',', $forSku));`
Делим по символу ',' и к каждому элементу применяем trim чтобы убрать пробелы в начале и конце.

https://github.com/anastasiaprakapchuk/AmastyLearningNewTechnology/blob/2df301cf33da20d9e475a94159fb4aa6509502d6/SecondAnastasiaModule/Observer/CheckAddProductObserver.php#L49
После этого стоит добавить проверку того, что промо продукт симпл.

Оценка 9

**Comment by anastasiaprakapchuk** [comment]
Спасибо, поправила замечания в task-7
